applications/ZeroPool.md

```diff
@@ -0,0 +1,147 @@
+
+- Samuele Landi Rust/blockchain developer
+
+https://github.com/samuelelandi 
+
+Samuele has experience in rust, blockchain, and polkadot development.
+
+Rust/blockchain developer builds relayer and maybe some parts of the pallet, which is needed for cryptography adapting for the final product.
+
+- Anton Pegov Frontend developer 
+
+https://github.com/antonpegov 
+
+Anton has experience in fronted (including fronted for Ethereum DApps).
+
+We need a separate position for the frontend developer because the scope of skills of other team members does not imply strong frontend skills. Anton covers this scope giving time for others to work hard on the codebase. 
+
+## **Team Website**
+
+- https://zeropool.network
+
+## **Team's experience**
+
+Igor is the tech lead of the team. Igor is responsible for core parts such as ZeroPool architecture and SNARKs. 
+
+Igor has great experience in research and product creations. He started as a researcher in physics at the same time he created some IT products for the b2b sector. Igor came to the blockchain as VP Eng in BANKEX Foundation and then became CTO there. Also, Igor has great experience in blockchain research (https://ethresear.ch/u/snjax/) and built [Fawkes-Crypto library](https://github.com/zeropoolnetwork/fawkes-crypto), providing development zkSNARKs as native applications in pure Rust. 
+
+Dmitry has a strong experience in rust. He worked on different IT projects before. 
+
+Anton is responsible for the frontend. He has perfect experience and skills in UX/UI. He worked for international IT companies. Also, he has experience in blockchain. He created a frontend for several dApps.
+
+Samuel has great experience in IT. He worked in application-directed cryptography. So he has deep knowledge of encryption, communication protocols, and coder of original communication protocols for encrypted communications. Also, Samuel worked on several blockchain projects with Polkadot (Substrate), Ethereum, and Bitcoin.
+
+Alexandra is responsible for product development and cooperation with other teams and companies.  Alexandra has great experience in IT projects and business development. Also, she is responsible for non-coding questions in the project. 
+
+## **Team Code Repos**
+
+- https://github.com/zeropoolnetwork
+- https://github.com/snjax
+- https://github.com/voidxnull 
+- https://github.com/samuelelandi 
+- https://github.com/antonpegov 
+
+ 
+
+## **Team LinkedIn Profiles**
+
+- https://www.linkedin.com/in/igorgulamov/<Igor Gulamov >
+- [https://www.linkedin.com/in/alexandra-gulamova/](https://www.linkedin.com/in/alexandra-gulamova-a4102a57/)<Alexandra Gulamova>
+
+## **Development Roadmap**
+
+* **Total Estimated Duration:** 4 months
+* **Full-time equivalent (FTE):**  3 FTE
+* **Total Costs:** 63000 DAI
+
+### Milestone 2 — Implement zkSNARK circuit and cryptography for private transaction
+
+* **Estimated Duration:** 2 months
```

Comment:
Please @snjax, submit an amendment. The duration should be updated to 5 months (which would be now) + however long you think you will still need.